Cargo.toml

```diff
@@ -9,6 +9,6 @@ edition = "2018"
 [dependencies]
 
 [dependencies.sdl2]
-version = "0.30.0"
+version = "0.34"
```

Comment:
I can remove this if you want, wasn't really intentional to change that, just a consequence of trying to resolve build issues.